src/widgetastic/ouia/__init__.py

```diff
@@ -23,7 +23,7 @@ class OUIABase:
     """
 
     ROOT = ParametrizedLocator(
-        ".//*[@data-ouia-component-type={@component_type}{@component_id_suffix}]"
+        ".//*[contains(@data-ouia-component-type,{@component_type}){@component_id_suffix}]"
```

Comment:
I think we should not try to use such a flexible approach. 
PF4 OR PF5 provides identity that's what purpose of switching to OUIA. so I would suggest using a locator for now. 

@mshriver your opinion on it.

Reply:
Hmm, but we're doing something similar already with the regular widgets in widgetastic.patternfly5 where they are backwards compatible with PF4 thanks to removing the `pf` part of the locator. How would this be different?

Reply:
Agreed, but we did it under the widget, to initialize widgets we have options like locator which we can customize but for OUIA component type we hardcode intentionally. somewhere we need to move to pf5.

Its my personal opinion so if everyone ok with this then let'ss go with it.

Reply:
@digitronik I understand your concern here, but I'm leaning towards accepting this approach.

In practice teams seldom care what actual version of PF is being used. In cases where someone wants to be exact with the data-ouia-component-type attribute they can override `OUIA_COMPONENT_TYPE` of a given wt.pf class, and even `ROOT` to use an exact match.

I think a holistic approach would use `substring-before` and `substring-after` to split the type attribute on the slash and separately check the version and type.

Longer term, given our desire to make the versions of wt.pf backwards compatible, I think we need to consolidate back into the wt.pf library.  I think we can leverage the wt.core `VersionPick` class here not against the target application but against the PF version of the component as read from the DOM where the components have different locators or behavior that needs to be implemented.

Reply:
there's a slight footgun with this approach, which is matching a component name substring.  Matching `/DropdownItem` in the DOM because you're using a widget with `OUIA_COMPONENT_TYPE="/Dropdown"` will be a headache.

Reply:
> there's a slight footgun with this approach, which is matching a component name substring. Matching `/DropdownItem` in the DOM because you're using a widget with `OUIA_COMPONENT_TYPE="/Dropdown"` will be a headache.

What do you mean? Like that it could match a different dropdown items than you want? Idk what "DOM" is.

Reply:
yep, but the risk is very minimal because the type isn't the only thing we're matching (as you raised in our discussion outside of the PR).